netcode.c

```diff
@@ -3861,9 +3923,9 @@ void skillz_clear_matches( struct netcode_server_t * server )
 {
     skillz_match_t * current_match, * tmp;
 
-    HASH_ITER( hh, server->matches, current_match, tmp )
+    HASH_ITER( hh, server->skillz_matches, current_match, tmp )
```

Comment:
Do you ever free the hash?

Reply:
The hash is freed when the last pointer is removed.  Its a handy library.